### PR TITLE
ARIA-166 Update README file

### PR DESCRIPTION
**Pull request by ran-z**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,198 +4,111 @@ ARIA
...
+![Python Versions](https://img.shields.io/pypi/pyversions/ariatosca.svg?style=plastic)
+![Wheel](https://img.shields.io/pypi/wheel/ariatosca.svg?style=plastic)
+![Contributors](https://img.shields.io/github/contributors/apache/incubator-ariatosca.svg?style=plastic)
+[![Open Pull Requests](https://img.shields.io/github/issues-pr/apache/incubator-ariatosca.svg?style=plastic)](https://github.com/apache/incubator-ariatosca/pulls)
+[![Closed Pull Requests](https://img.shields.io/github/issues-pr-closed-raw/apache/incubator-ariatosca.svg?style=plastic)](https://github.com/apache/incubator-ariatosca/pulls?q=is%3Apr+is%3Aclosed)
 
 
-[ARIA](http://ariatosca.org/) is a minimal TOSCA orchestrator, as well as a platform for building
-TOSCA-based products. Its features can be accessed via a well-documented Python API.
+[ARIA](http://ariatosca.incubator.apache.org/) is a an open-source, lightweight, library and CLI for orchestration that open-source projects can consume to build TOSCA-based orchestration solutions for resources and services orchestration. It supports NFV and hybrid Cloud scenarios.
```
Comment (AviaE) [inline README.md]: Consider removing the second occurrence of `orchestration`.

File: README.md [hunk 2]
```diff
@@ -4,198 +4,111 @@ ARIA
...
-The ARIA API documentation always links to the relevant section of the specification, and likewise
-we provide an annotated version of the specification that links back to the API documentation.
+ARIA is [available on PyPI](https://pypi.python.org/pypi/ariatosca).    
 
+ARIA requires Python 2.6/2.7. Python 3+ is not currently supported.
 
-Quick Start
------------
+ARIA itself is in a `wheel` format compatible with all platforms. 
+Some dependencies, however, might require compilation (based on a given platform), and therefore possibly some system dependencies are required as well.
```
Comment (AviaE) [inline README.md]: Maybe move these 'disclaimers' to the bottom, under troubleshooting or something similar.

File: README.md [hunk 3]
```diff
@@ -4,198 +4,111 @@ ARIA
...
-You can provide inputs as JSON, overriding default values provided in the blueprint
+Getting Started
+---------------
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml --inputs='{"openstack_credential": {"user": "username"}}'
+This section will describe how to run a simple "Hello World" example.
 
-Instead of providing them explicitly, you can also provide them in a file or URL, in either JSON or
-YAML. If you do so, the value must end in ".json" or ".yaml":
+First, provide ARIA with your service-template, naming it `my-service-template`:
```
Comment (AviaE) [inline README.md]: 'your service template' --> 'the ARIA hello world service template'
'naming it [...]' --> 'and name it, e.g. `my-service-template`'

File: README.md [hunk 4]
```diff
@@ -4,198 +4,111 @@ ARIA
...
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml --inputs=blueprints/tosca/node-cellar/inputs.yaml
+	aria service-templates store examples/hello-world/helloworld.yaml my-service-template
+	
+Now create a service based on this service-template, naming it `my-service`:
```
Comment (AviaE) [inline README.md]: 'naming it' --> 'and name it, e.g. my-service'

File: README.md [hunk 5]
```diff
@@ -4,198 +4,111 @@ ARIA
...
----
+<br>
+You should now have a simple web-server running on your local machine.
+You can try visiting http://localhost:9090 to view your deployed application.
 
-Though ARIA is fully exposed as an API, it also comes with a CLI tool to allow you to work from the
-shell:
+To uninstall and clean your environment, follow these steps:
 
-	aria parse blueprints/tosca/node-cellar/node-cellar.yaml instance
```
Comment (AviaE) [inline README.md]: I just noticed that `executions start uninstall` looks weird. Maybe it would be better if we change `start` to `execute`?
Then under `executions`, we'll have `execute`, `show`, `list`. Just a thought =)

File: README.md [hunk 6]
```diff
@@ -4,198 +4,111 @@ ARIA
...
+    aria executions start uninstall -s my-service
```
Comment (AviaE) [inline README.md]: Maybe add comments to these three lines, like:
`# execute the uninstall workflow`
`# delete the service`
`# delete the service template`

File: README.md [hunk 7]
```diff
@@ -4,198 +4,111 @@ ARIA
...
+Contribution
+------------
 
-Additionally, The CLI tool lets you specify the complete classname of your own custom consumer to
-chain at the end of the default consumer chain, after `instance`.
+You are welcome and encouraged to participate and contribute to the ARIA project.
 
-Your custom consumer can be an entry point into a powerful TOSCA-based tool or application, such as
-an orchestrator, a graphical modeling tool, etc.
+Please see our guide to [Contributing to ARIA](https://cwiki.apache.org/confluence/display/ARIATOSCA/Contributing+to+ARIA).
```
Comment (AviaE) [inline README.md]: This is like the first time that Apache is mentioned. Shouldn't it be mentioned before?

File: README.md [hunk 8]
```diff
@@ -4,198 +4,111 @@ ARIA
...
-	. env/bin/activate
-	git clone http://git-wip-us.apache.org/repos/asf/incubator-ariatosca.git ariatosca
-	cd ariatosca
-	pip install -e .
+* [ARIA homepage](http://ariatosca.incubator.apache.org/)
+* [ARIA wiki](https://cwiki.apache.org/confluence/display/AriaTosca)
+* [Issue tracker](https://issues.apache.org/jira/browse/ARIA)
 
-To run tests:
+* Dev mailing list: dev@ariatosca.incubator.apache.org
```
Comment (AviaE) [inline README.md]: Add quick explanation here on how to subscribe to the mailing lists, and for info, there's that link...

Comment (ran-z) [inline README.md]: :+1:

File: README.md (hunk 6 above)
Comment (ran-z) [inline README.md]: i'd rather not - adding "bash comments" would make this subsection inconsistent with the installation subsection, while making it consistent (i.e. having separate text which explains each and every command) seems like an overkill in this case.

File: README.md (hunk 5 above)
Comment (ran-z) [inline README.md]: Not my circus, not my monkeys.

File: README.md (hunk 7 above)
Comment (ran-z) [inline README.md]: :+1:

File: README.md (hunk 2 above)
Comment (ran-z) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (ran-z) [inline README.md]: meh dunno ~_~

File: README.md (hunk 3 above)
Comment (ran-z) [inline README.md]: :+1:

File: README.md (hunk 4 above)
Comment (ran-z) [inline README.md]: :+1:

File: README.md [hunk 9]
```diff
@@ -12,32 +12,23 @@ ARIA
 [![Closed Pull Requests](https://img.shields.io/github/issues-pr-closed-raw/apache/incubator-ariatosca.svg?style=plastic)](https://github.com/apache/incubator-ariatosca/pulls?q=is%3Apr+is%3Aclosed)
 
 
-[ARIA](http://ariatosca.incubator.apache.org/) is a an open-source, lightweight, library and CLI for orchestration that open-source projects can consume to build TOSCA-based orchestration solutions for resources and services orchestration. It supports NFV and hybrid Cloud scenarios.
+What is ARIA?
+----------------
+
+[ARIA](http://ariatosca.incubator.apache.org/) is a an open-source, [TOSCA](https://www.oasis-open.org/committees/tosca/)-based, lightweight library and CLI for orchestration and for consumption by projects building TOSCA-based solutions for resources and services orchestration.
```
Comment (AviaE) [inline README.md]: building=creating?

File: README.md [hunk 10]
```diff
@@ -12,32 +12,23 @@ ARIA
...
 
 ARIA can be utilized by any organization that wants to implement TOSCA-based orchestration in its solutions, whether a multi-cloud enterprise application, or an NFV or SDN solution for multiple virtual infrastructure managers.
 
-With ARIA, you can utilize TOSCA's cloud portability out-of-the-box, to test and run your applications, from template to deployment. 
+With ARIA, you can utilize TOSCA's cloud portability out-of-the-box, to develop, test and run your applications.
```
Comment (AviaE) [inline README.md]: why did you remove 'from template to deployment'?

File: README.md [hunk 11]
```diff
@@ -47,25 +38,39 @@ To install ARIA from source, download the source tarball from [PyPI](https://pyp
...
+ARIA requires Python 2.6/2.7. Python 3+ is currently not supported.
 
 
 Getting Started
 ---------------
 
 This section will describe how to run a simple "Hello World" example.
 
-First, provide ARIA with your service-template, naming it `my-service-template`:
+First, provide ARIA with the ARIA "hello world" service-template, and name it - e.g. `my-service-template`:
```
Comment (AviaE) [inline README.md]: The parenthesis version of e.g. was better.

File: README.md (hunk 9 above)
Comment (ran-z) [inline README.md]: nope

File: README.md (hunk 10 above)
Comment (ran-z) [inline README.md]: :+1:

File: README.md (hunk 11 above)
Comment (ran-z) [inline README.md]: :+1:
